src/controllers/movieController.ts

```diff
@@ -0,0 +1,27 @@
+import {MovieRepository} from "../repositories/movieRepository";
+import {MovieModel} from "../models/movieModel";
+
+export class MovieControllers {
```

Comment:
Yo lo suelo llamar:
```suggestion
export class MovieController {
```
o MoviesController